include/verilated_cov.h

```diff
@@ -75,6 +75,58 @@ class VerilatedCovImp;
         ccontextp->_insertp("hier", name, __VA_ARGS__); \
     } while (false)
 
+static inline void VL_COV_TOGGLE_CHG_S_I(const int width, uint32_t* cov, const IData newData,
+                                         const IData oldData) {
+    for (int i = 0; i < width; ++i) *(cov + i) += ((newData ^ oldData) >> i) & 1;
+}
+
+static inline void VL_COV_TOGGLE_CHG_S_Q(const int width, uint32_t* cov, const IData newData,
+                                         const IData oldData) {
+    for (int i = 0; i < width; ++i) *(cov + i) += ((newData ^ oldData) >> i) & 1;
+}
+
+static inline void VL_COV_TOGGLE_CHG_S_W(const int width, uint32_t* cov, WDataInP newData,
+                                         WDataInP oldData) {
+    for (int i = 0; i < VL_WORDS_I(width); ++i) {
+        const EData changed = newData[i] ^ oldData[i];
+        if (changed) {
+            for (int j = 0; j < width - i * 32; ++j) *(cov + i * 32 + j) += (changed >> j) & 1;
+        }
+    }
+}
+
+static inline void VL_COV_TOGGLE_CHG_M_I(const int width, std::atomic<uint32_t>* cov,
+                                         const IData newData, const IData oldData) {
```

Comment:
```suggestion
static inline void VL_COV_TOGGLE_CHG_MT_I(const int width, std::atomic<uint32_t>* covp,
                                         const IData newData, const IData oldData) VL_MT_SAFE {
```

_MT as otherwise looks like a type letter (use _ST for others above).  Also add VL_MT_SAFE tag.  `covp` as is a pointer.